tools/packaging/kernel/configs/fragments/x86_64/sev/sev.conf

```diff
@@ -1,12 +1,9 @@
 # AMD Secure Encrypted Virtualization (SEV)
  
 CONFIG_AMD_MEM_ENCRYPT=y
-CONFIG_AMD_MEM_ENCRYPT_ACTIVE_BY_DEFAULT=y
-CONFIG_CRYPTO_DEV_SP_PSP=y
-CONFIG_CRYPTO_DEV_CCP=y
-CONFIG_SECURITYFS=y
-CONFIG_VIRT_DRIVERS=y
 CONFIG_EFI=y
 CONFIG_EFI_SECRET=m
+CONFIG_EFI_STUB=y
+CONFIG_MODULES=y
```

Comment:
Please, keep this sorted alphabetically.

Reply:
Just out of curiosity. If kata always use direct kernel boot, what does efi stub kernel use for?

Reply:
https://www.kernelconfig.io/CONFIG_EFI_STUB?q=CONFIG_EFI_STUB&kernelversion=5.19

This kernel feature allows a bzImage to be loaded directly
by EFI firmware without the use of a bootloader.

Perhaps it should be added to a different conf?

Reply:
@fidencio, pushed the alphabetization.

Reply:
AFAIK, there is no kernel inside kata image. Am I right? @fidencio . So, where does firmware load the EFI stub kernel?

Reply:
My understanding is:

In direct boot (with `-kernel`), QEMU passes the content of the kernel (and initrd, and cmdline) to the guest via the fw_cfg interface. OVMF (firmware) reads the kernel from that interface, places it in memory, and jumps to the kernel's EFI stub.

Since SEV (and TDX) must have OVMF (or TDVF), the kernels for these guests must have `CONFIG_EFI_STUB=y`.